doc/src/learn/single-writer-apps.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Sui Single-Writer-Friendly (SWF) apps
+---
+
+This page lists applications that can work in the single-writer model defined as [simple transactions](how-sui-works.md#simple-transactions) in Sui. Some proposals that typically require shared objects have been transformed to variants that require only a shared object as a final step and not for every action, such as voting and lotteries, DeFi Oracle price quotes, etc.
+
+1. Regular peer-to-peer (p2p) transactions ([see how to create a new Coin with just 7 lines of Sui Move code](https://www.linkedin.com/posts/chalkiaskostas_startup-smartcontract-cryptocurrency-activity-6946006856528003072-CvI0)).
+1. Confidential p2p Txs: same as FastPay but with pedersen commitments to hide amounts transferred; this still ensures input amount = output amount - we can set amount limits, i.e., N transfers up to $1,000 can be confidential.
+1. Public bulletin board; users store only publicly accessed data, files, links, metadata.
+1. Proof of existence: the above with time-stamped documents - it can be extended to support commitment proof of existence (publish your hash then reveal).
+1. Private decentralized repository (users store private files, encrypted under their pub keys); the pubKeys can be NFTs.
+1. Extend the above for selected disclosure CV (resume) repository, Uni degrees repository.
+1. Decentralized or conventional Certificate Authority. Authorities publish their signatures over certs, they can revoke any time (easier revocation).
+1. Messaging service: apps, Oracles and Internet of things (IoTs) exchanging messages. Sui is probably the best platform for any messaging protocol, as typically each response and message can be encoded with a single writer NFT.
+1. Extend the above to social networks (Suitter) - note that each post is a single-writer NFT. See a [smart contract implementation of a fully functional decentralized Twitter with just 50 lines of Sui Move code](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/nfts/sources/chat.move).
```

Comment:
replace with "Extend the above to social networks; note that each post can be encoded as a single-writer NFT."